CONTRIBUTING.md

```diff
@@ -0,0 +1,326 @@
+- **Domain Separation**: Respect the Core/Edge/Content domain boundaries
+
+### Python Style
+
+- Follow PEP 8 guidelines (automatically enforced by Black)
+- Use descriptive variable and function names
+- Keep functions focused and small
+- Prefer composition over inheritance
+
+## Testing
+
+### Running Tests
+
+```bash
+# Run all tests
+pytest
+
+# Run specific test file
+pytest analyzers/hashtags/test_hashtags_analyzer.py
+
+# Run with verbose output
+pytest -v
+```
+
+### Test Guidelines
+
+- Write tests for all new functionality
+- Place tests co-located with the modules they test
+- Use the testing framework provided in the `testing/` module
+- Include test data in `test_data/` directories within analyzer modules
+- Ensure tests are fast and reliable
+
+### Test Types
+
+- **Unit Tests**: Test individual functions and methods
+- **Integration Tests**: Test analyzer workflows
+- **End-to-End Tests**: Test complete user workflows
+
+## Submitting Changes
+
+### Pull Request Guidelines
+
+1. **Target Branch**: Always target `develop`, never `main`
+2. **Description**: Provide a clear description of changes
+3. **Testing**: Include test results and coverage information
+4. **Documentation**: Update relevant documentation
+5. **Breaking Changes**: Clearly mark any breaking changes
+
+### PR Checklist
+
+- [ ] Code follows project coding standards
+- [ ] Tests added for new functionality
+- [ ] All tests pass
+- [ ] Documentation updated
+- [ ] Commit messages are clear and descriptive
+- [ ] PR targets `develop` branch
+- [ ] Pre-commit hooks pass
+
+### Commit Message Format
+
+Use conventional commit format:
+
+```markdown
+type(scope): brief description
+
+Detailed explanation of the change (if needed)
+
+- List specific changes
+- Include any breaking changes
+- Reference issues if applicable
+
+Fixes #123
+```
+
+Types: `feat`, `fix`, `docs`, `style`, `refactor`, `test`, `chore`
+
+## Code Review Process
+
+1. **Automated Checks**: CI/CD pipeline runs tests and quality checks
+2. **Manual Review**: Project maintainers review code and architecture
+3. **Feedback**: Address any requested changes
+4. **Approval**: Maintainer approval required before merge
+
+## Release Process
+
+- Features are merged into `develop`
+- When `develop` is stable, it's merged into `main` for release
+- Releases follow semantic versioning
+
+## Getting Help
+
+### Documentation
+
+- **Development Guide**: [docs/dev-guide.md](docs/dev-guide.md)
+- **AI Context Docs**: `docs/ai-context/` directory
+- **Architecture Overview**: `docs/ai-context/architecture-overview.md`
+- **Symbol Reference**: `docs/ai-context/symbol-reference.md`
+
+### Community Support
+
+- **Slack**: Join the [Civic Tech DC Slack workspace](https://civictechdc.slack.com)
+- **Issues**: Use GitHub Issues for bug reports and feature requests
+- **Discussions**: Use GitHub Discussions for questions and general discussion
```

Comment:
Not sure, we're using GH Discussions much atm, I think it's mostly Slack for general discussions, perhaps this can be dropped?